Lib/test/test_turtle.py

```diff
@@ -440,6 +445,17 @@ def test_showturtle_hideturtle_and_isvisible(self):
         tpen.showturtle()
         self.assertTrue(tpen.isvisible())
 
+    def test_teleport(self):
+
+        tpen = turtle.TPen()
+
+        tpen.penup()
+        tpen.teleport(100, 100)
+        self.assertTrue(not tpen.isdown())
```

Comment:
assertFalse

Reply:
I've changed this in the latest commit, thanks!